pkgs/misc/cups/libppd.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, cups
+, libcupsfilters
+, ghostscript
+, pkg-config
+, poppler_utils
+, mupdf # mutool
```

Comment:
This should be moved to the line where the package is actually used, so that it is in context on changes.

Reply:
resolved (mutool is now specified explicitly, so the use of mupdf should be clear now)